### ISSUE TITLE
Feature Request: Skills for perfect fit

### ISSUE THREAD

**Issue by HeidiVinter** [pyfa-org/Pyfa#753]
Is it possible to get a Skills for the perfect fit. 

In a way that it is similar to the required skills, but shows all skills affecting the current fit that would be would need to be trained to V to get the perfect skills for that fit. ie. best resistances, most cap remaining, fastest speed, highest DPS, most fitting space left over.

**Comment by blitzmann** [comment]
So you're basically looking for a list of all skills used, and then have them at lvl 5? What would the use case be?

**Comment by HeidiVinter** [comment]
Yes basically, but have them compared against your current skills. For training alts into specific ships/fits

**Comment by minlexx** [comment]
So, is it like ship certificate level 5, not only for ship, but for all modules used in fit?

**Comment by HeidiVinter** [comment]
too easy, thanks for the response

**Comment by Rylleyryelle** [comment]
Even on handles fitting skills only, not all skills that affect the ship. This would actually be super useful. Just like every skill that's utilised in the fit into a single list that you could then import to evemon

**Comment by Rylleyryelle** [comment]
Even on = evemon. Typing on phone and it hates me

**Comment by Ebag333** [comment]
Something I've brought up before are ship masteries.  I'm 92.3% sure it's in Crest, and we could probably expose it fairly easily and mark which masteries you have just like CCP does in game.

That's a pretty low hanging fruit though.
